test/common/router/router_test.cc

```diff
@@ -2253,7 +2253,9 @@ TEST_F(RouterTest, AltStatName) {
 
 TEST_F(RouterTest, Redirect) {
```

Comment:
I don't see any tests in here for actually setting stream info. Can you add them?

Reply:
@mattklein123 I have added a couple of checks for this. I also made a separate commit to with changes in access logs - this will cover the tests for route name in stream info. Let me know if this works - thanks!